src/controllers/ReviewPenaltyController.ts

```diff
@@ -0,0 +1,101 @@
+import { SessionMiddleware } from 'ch-node-session-handler';
+import { Penalty, PenaltyList } from 'ch-sdk-node/dist/services/lfp';
+import { controller } from 'inversify-express-utils';
+import { SafeNavigationBaseController } from './SafeNavigationBaseController';
+
+import { AuthMiddleware } from 'app/middleware/AuthMiddleware';
+import { Appeal } from 'app/models/Appeal';
+import { OTHER_REASON_DISCLAIMER_PAGE_URI, PENALTY_DETAILS_PAGE_URI, REVIEW_PENALTY_PAGE_URI } from 'app/utils/Paths';
+import { Navigation } from 'app/utils/navigation/navigation';
+
+const template = 'review-penalty';
+const navigation: Navigation = {
+    next: () => OTHER_REASON_DISCLAIMER_PAGE_URI,
+    previous: () => PENALTY_DETAILS_PAGE_URI
+};
```

Comment:
Think its better to stick to the convention we have been using across all controllers

```suggestion
const navigation = {
    previous(): string {
        return PENALTY_DETAILS_PAGE_URI;
    },
    next(): string {
        return OTHER_REASON_DISCLAIMER_PAGE_URI;
    }
};
```